web/src/engine/websites/ConstellarScans.ts

```diff
@@ -0,0 +1,82 @@
+import { Tags } from '../Tags';
+import icon from './ConstellarScans.webp';
+import { type Chapter, DecoratableMangaScraper, Page } from '../providers/MangaPlugin';
+import * as MangaStream from './decorators/WordPressMangaStream';
+import * as Common from './decorators/Common';
+import { Fetch, FetchRequest } from '../FetchProvider';
+
+type TSREADER = {
+    sources: {
+        images : string[]
+    }[]
+}
+
+@MangaStream.MangaCSS(/^{origin}\/manga\/[^/]+\/$/)
+@MangaStream.MangasSinglePageCSS()
+@MangaStream.ChaptersSinglePageCSS()
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('constellarscans', 'Constellar Scans', 'https://constellarcomic.com', Tags.Media.Manhwa, Tags.Media.Manhua, Tags.Media.Manga, Tags.Language.English, Tags.Source.Scanlator, Tags.Rating.Pornographic);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override async FetchPages(chapter: Chapter): Promise<Page[]> {
+        const uri = new URL(chapter.Identifier, this.URI);
+        const request = new FetchRequest(uri.href);
+        const response = await Fetch(request);
+        const data = await response.text();
+        let tsrundata = '';
+        let ts_reader: TSREADER = undefined;
+
+        //get renamed ts_reader
+        try {
```

Comment:
Maybe it would be easier  to just get the images via JS script `div#reader_area img`

Reply:
IIRC we dont have image url at all. Images src are placeholder .